templates/kubernetes/overlays/production/auth.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: oathkeeper.ory.sh/v1alpha1
```

Comment:
Could you add a doc to each of these so people know what they are for? Especially the backend endpoints which are the ones a developer is more likely to need to change in the future.
Actually, I wonder if kratos-public and kratos-form-data should be set up in the kubernetes terraform instead, since they aren't really specific to the app.. And maybe we even want a different subdomain for it? like `auth.<% index .Params `productionHostRoot` %>`?
Might be worth doing some testing though, because it depends on how they have set up the session cookie. If it's tied to the subdomain we'll have to do it this way otherwise it won't be able to read the cookie.

Reply:
thats true, they dont really have much to do with the backend itself 🤔   

as in Oathkeeper will listen from 2 ingresses? then proxy to the same place after? 
yeah the cookie/redirection stuff may cause issues